vars/uploadMinSnapshotsToS3.groovy

```diff
@@ -28,9 +28,12 @@ void call(Map args = [:]) {
     }
 
     withAWS(role: "${ARTIFACT_PROMOTION_ROLE_NAME}", roleAccount: "${AWS_ACCOUNT_ARTIFACT}", duration: 900, roleSessionName: 'jenkins-session') {
```

Comment:
Instead of doing this, we can use the upload library that was recently created

Reply:
This is a bug fix PR not a feature enhancement PR.
I disagree this is the reason to block this PR.
We can have a separate PR later if we want to improve the process.

Reply:
Also that lib is only having one mode which is using file to file.
It does not support wildcard push as of yet.
So not really suitable for us to use it now.

Reply:
Thanks.